tests/Integration/FailedRequestRepositoryTest.php

```diff
@@ -2,6 +2,7 @@
 
 namespace Tests\Integration;
 
+use DateTime;
 use Towa\GebruederWeissWooCommerce\Plugin;
 use Towa\GebruederWeissWooCommerce\FailedRequestQueue\FailedRequest;
 use Towa\GebruederWeissWooCommerce\FailedRequestQueue\FailedRequestRepository;
```

Comment:
regarding line 24 in this file: i think this row should somehow get deleted after the test

Reply:
The wordpress testing framework cleans the database after each individual test. So all changes made during a test will be undone.